index.bs

```diff
@@ -387,8 +388,25 @@ The [=agent certificate=] must have the following characteristics:
 <tbody>
 <tr>
   <td>`secp256r1_sha256`</td>
-  <td>`301306072a8648ce3d020106082a8648ce3d030107`</td>
-  <td>`300a06082a8648ce3d040302`</td>
+  <td>`secp256r1`
+     <p>
+      Object Identifiers:
+      <ol>
+        <li>`1.2.840.10045.2.1` (ECC)</li>
+        <li>`1.2.840.10045.3.1.7` (ECDSA P256)</li>
+      </ol>
+      DER representation: `301306072a8648ce3d020106082a8648ce3d030107`
+      </p>
+  </td>
+  <td>`ecdsaWithSha256`
```

Comment:
I suppose the object name is a reference to the definition in RFC5758. The RFC seems to use hyphens for this one:

```suggestion
  <td>`ecdsa-with-SHA256`
```